portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -202,12 +202,13 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 							}
 
 							for (int i = pagesIteratorBegin; i <= pagesIteratorEnd; i++) {
+								String curParamUrl = HttpUtil.setParameter(url + urlAnchor, namespace + curParam, i);
```

Comment:
Minor SF: variable should be `curParamURL`. Even better, this can be inline, so we avoid the variable.